server/models/post.js

```diff
@@ -1,12 +1,14 @@
 const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
+const cuid = require('cuid');
 
 const postSchema = new Schema({
+    cuid: { type: 'String', default: cuid, required: true },
```

Comment:
this could be implemented directly like this (overrides _id generation):
_id: { type: 'String', default: cuid, required: true },

with this option we can use the followin code into Post schema to relate schemas:
user: {type: 'String', ref "User" }

but i left it as it was because didn't want to break the front cuid existent logic.